sklearn/preprocessing/imputation.py

```diff
@@ -374,3 +385,352 @@ def transform(self, X):
             X[coordinates] = values
 
         return X
+
+
+class MICEImputer(BaseEstimator, TransformerMixin):
+    """Basic implementation of MICE package from R.
+    This version assumes all of the features are Gaussian.
+
+    Parameters
+    ----------
+    missing_values : int or "NaN", optional (default="NaN")
+        The placeholder for the missing values. All occurrences of
+        missing_values will be imputed. For missing values encoded as
+        np.nan, use the string value "NaN".
+
+    imputation_order : str, optional (default="monotone")
+        The order in which the features will be imputed.
+
+        - "monotone" - From features with fewest missing values to most.
+        - "revmonotone" - From features with most missing values to fewest.
+        - "roman" - Left to right.
+        - "arabic" - Right to left.
+        - "random" - A random order for each round.
+
+    n_imputations : int, optional (defaults = 100)
+        Number of MICE rounds to perform the results of which will be
+        used in the final average.
+
+    n_burn_in : int, optional (default = 10)
+        Number of initial MICE rounds to perform the results of which
+        will not be returned.
+
+    n_nearest_features : int, optional (default = np.infty)
```

Comment:
(default = np.infty) -> (default=np.infty)

Reply:
infinity is weird. You should use probably `None` which mean that all features will be used.